docs/reference/migration/migrate_7_0/migrate_to_java_time.asciidoc

```diff
@@ -0,0 +1,350 @@
+[[migrate_to_java_time]]
+=== Java time migration guide
+
+With 7.0, {es} switch to java time from joda time for date-related parsing,
+formatting, and calculations. This guide is designed to help you determine
+if your cluster is impacted and, if so, prepare for the upgrade.
+
+Joda style indices are supported in 6.8, but indices created with joda-style patterns in 6.x are also allowed to be used in 7.0
+All new indices created in 7.x are expected to be using java-time patterns.
+
+We advice to prepare for this migration in 6.8, but it will also work if you decide to perform it in version 7.x
```

Comment:
While you can update any existing joda-time patterns in 7.x, we recommend you make these change 6.8 if possible.